server/server.js

```diff
@@ -1,14 +1,16 @@
 var express = require('express');
 
```

Comment:
I wouldn't use enters between variables, especially not when they belong together (both requires).